### ISSUE TITLE
Fatal GuzzleHttp error

### ISSUE THREAD

**Issue by adrian-stein** [br33f/php-GA4-Measurement-Protocol#39]
I have set all the code up and try to send either the send or sendDebug method and it throws the below error.

`Fatal error: Declaration of GuzzleHttp\Psr7\Uri::withScheme($scheme) must be compatible with Psr\Http\Message\UriInterface::withScheme(string $scheme) in \vendor\guzzlehttp\psr7\src\Uri.php on line 14`

Any ideas? I am running php 7.1.

**Comment by br33f** [comment]
Hi,
what is your installed GuzzleHttp version ?

**Comment by adrian-stein** [comment]
"name": "guzzlehttp/guzzle",
"version": "6.5.x-dev",

**Comment by adrian-stein** [comment]
Hey there,

composer.lock says these version.

```
"name": "guzzle/guzzle",
"version": "dev-master",

"name": "guzzlehttp/guzzle",
"version": "6.5.x-dev",

"name": "guzzlehttp/promises",
"version": "1.5.x-dev",

"name": "guzzlehttp/psr7",
"version": "1.9.x-dev",
```

**Comment by adrian-stein** [comment]
I also just tried setting guzzlehttp/guzzle to 6.5.5 and get the same issue

**Comment by br33f** [comment]
Please try guzzlehttp/guzzle version ^7.0.0.
I hope it will work.

**Comment by adrian-stein** [comment]
That works. it conflitcs with other plguins I have but at least it can install this one and try to work out the other issues. Thanks

**Comment by adrian-stein** [comment]
Nope. I still get the same error. have you got it working with similar settings?

**Comment by adrian-stein** [comment]
@br33f any word on this thanks??

**Comment by br33f** [comment]
@adrian-stein I have no idea i can not reproduce this error.
